listenbrainz/webserver/static/js/src/Loader.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import { faSpinner } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+
+type LoaderProps = {
+  isLoading: boolean;
+};
+
+const Loader: React.SFC<LoaderProps> = (props) => {
+  // eslint-disable-next-line react/prop-types
```

Comment:
Do we need this rule? Can we disable it entirely?

Reply:
Actually we already have disabled it entirely, https://github.com/metabrainz/listenbrainz-server/blob/master/.eslintrc.js#L27. However it does not seem to work in this case. Some bug in `ESLint` I guess.